misc/ls-extensions/modules/json-to-record-converter/src/main/java/io/ballerina/converters/JsonToRecordRequest.java

```diff
@@ -78,4 +80,12 @@ public boolean getForceFormatRecordFields() {
     public void setForceFormatRecordFields(boolean forceFormatRecordFields) {
         this.forceFormatRecordFields = forceFormatRecordFields;
     }
+
+    public String getFilePath() {
+        return filePath;
+    }
```

Comment:
Since the file uri is optional in the request, shall we return an `Optional<String>` in here instead, so that the getter will be self explanatory?

Reply:
Yeah, that would make sense 👍

Reply:
Serialization of `string` to `Optional<String>` fails as it is not supported. so we have to handle `null` manually.